wynnset/tapestry-wp
# 604 - Fix Docker Plugins Permissions

Closes #604 

Change file structure of local environment to use plugins folder in persistent volume rather than bind mount
Fixes ownership issues of files inside container, and allows Apache to write to `/var/www/html` folder
Removes now unnecessary scripts for permission management

Steps to test:
- Launch docker containers
- Reinstall plugins using WP graphical interface and ensure there are no errors
- Also try uploading a file to a tapestry as changes effect the `uploads` folder as well
